src/olympia/bandwagon/views.py

```diff
@@ -703,14 +704,35 @@ def get_addons_queryset(self):
         return qs.order_by(sort)[:limit]
 
 
+class TranslationAwareOrderingAliasFilter(OrderingAliasFilter):
+    def filter_queryset(self, request, queryset, view):
+        ordering = self.get_ordering(request, queryset, view)
+
+        if len(ordering) > 1:
+            # We can't support multiple orderings easily because of
+            # how order_by_translation works.
+            raise serializers.ValidationError(
+                'You can only specify one "sort" argument. Multiple '
+                'orderings are not supported')
+
+        order_by = ordering[0]
+
+        if order_by.lstrip('-') == 'addon__name':
+            prefix = '-' if order_by.startswith('-') else ''
+            return order_by_translation(queryset, prefix + 'name', Addon)
```

Comment:
I think you can skip some of this string manipulation by dropping ` 'name': 'addon__name' ` from `ordering_field_aliases` in `CollectionAddonViewSet` so it's just `name` anyway.

Reply:
Good idea, fixed in 631db63